index.bs

```diff
@@ -1823,7 +1823,51 @@ credential. It is intended primarily for [RPS] that wish to tightly control the
                 82
     </pre>
 
+## Location Extension ## {#uvi-location}
 
+: Extension identifier
+:: `webauthn_loc`
+
+: Client argument
+:: The Boolean value `true` to indicate that this extension is requested by the [RP].
+
+: Client processing
+:: None, except default forwarding of client argument to authenticator argument.
+
+: Authenticator argument
+:: The Boolean value `true`, encoded in CBOR (major type 7, value 21).
+
+: Authenticator processing
+:: If the <a>authenticator</a> does not support the extension, then the authenticator MUST ignore the extension request.  
+    If the <a>authenticator</a> accepts the extension, then the authenticator SHOULD only add this extension data to a packed 
+    attestation or assertion.
+
+: Authenticator data
+:: If the <a>authenticator</a> accepts the extension request, then authenticator data SHOULD provide location data in the form of a 
+    CBOR-encoded map, with the first value being the extension identifier and the second being an array of returned values.  The array 
+    elements SHOULD be derived from (key,value) pairings for each location attribute that the <a>authenticator</a> supports. The following is 
+    an example of authenticator data where the returned array is comprised of a {longitude, latitude, altitude} triplet.
```

Comment:
We need to define whether altitude means "meters above mean sea level", "feet above MSL", "meters above WGS84 ellipsoid", "feet above WGS84 ellipsoid" or something else.